.github/workflows/ci.yaml

```diff
@@ -35,23 +35,26 @@ jobs:
       fail-fast: false
       matrix:
         # Test modern Odoo versions with latest Postgres version
-        pg_version: ["14"]
-        odoo_version:
-          - "16.0"
-          - "15.0"
-          - "14.0"
-          - "13.0"
+        pg_version: ["15"]
+        odoo_version: ["17.0"]
+        python_version: ["3.10"]
         include:
-          # Older odoo versions don't support latest postgres because of the adapter
-          - pg_version: "13"
+          # Older odoo versions don't support latest postgres and Python versions
+          - pg_version: "14"
+            odoo_version: "15.0"
+            python_version: "3.9"
+          - pg_version: "14"
+            odoo_version: "14.0"
+            python_version: "3.9"
+          - pg_version: "14"
             odoo_version: "13.0"
+            python_version: "3.9"
           - pg_version: "13"
-            odoo_version: "12.0"
-          - pg_version: "12"
-            odoo_version: "11.0"
+            odoo_version: "13.0"
```

Comment:
**thought:** I find it weird that you test odoo 13 twice and you don't run odoo 11, 12 or 16.

Other than that, I see no reason why builds for 17 get aborted.

Reply:
Ouch, my fault! I want it to test it just once for 17, 16, 15, 14 and 13, and remove official support for 11 and 12.